packages/eslint-plugin/src/rules/explicit-member-accessibility.ts

```diff
@@ -1,7 +1,52 @@
-import { TSESTree } from '@typescript-eslint/typescript-estree';
+import { TSESTree, AST_NODE_TYPES } from '@typescript-eslint/typescript-estree';
 import * as util from '../util';
 
-export default util.createRule({
+enum Check {
+  Yes,
+  No,
+  NoPublic,
+}
+
+interface NoPublic {
+  noPublic?: boolean;
+}
+
+type Override = boolean | NoPublic;
+
+/**
+ * Type guard to ease checking on which kind of override is being supplied
+ *
+ * @param {Override} [option]
```

Comment:
please delete the unnecessary jsdoc lines.
They are just noise if they don't have descriptions attached to them

Reply:
Done.